Automattic/wp-calypso
# Gutenboarding: Adiós, wizard 👋

#### Changes proposed in this Pull Request
In previously iterations of Aquire Intent screen, we've implemented a bunch of things for that are obsolete now. They were implemented to support early designs. The designs are simpler now so we could remove a lot of complexity while improving the UX.

##### Improvements:
- fixed 2 issues regarding suggestion selection mentioned in p1585212736030000-slack-gutenboarding so that:
  - tabbing to from vertical selection input when the suggestions are visible should work as expected (automatically selecting first Suggestion and filling the input field with its label).
  - pressing Enter key while focused on the vertical selection input is setting the current input value as a Vertical (like in Google Search 😁 cc: @dubielzyk) instead of selecting first item from the suggestion list.
- Use grey bottom border together with the animated placeholder when the vertical input is empty
- Use arrow at the end of the phrase _My site is about [vertical]_ when the vertical input is focused
- Hide Suggestion list when empty & focused to prevent [this issue](https://cloudup.com/c84w4Br0hvP)
- Focus vertical input at initialisation if there is no selected vertical

#### Testing instructions
* Go to https://hash-0b1d417eaec047290262031289c746ca74e41d00.calypso.live/gutenboarding 
* Write anything then press **Enter**. the user input should always be saved including: `Food`, `Restaurant` or any other existing vertical that is not on the first position in the list of suggestions.
* While the suggestion list is visible (for a matching query), pressing **Tab** should select the first item and advance to Site Title input.
* Check the existing functionality + the improvements mentioned above

Fixes #40142

Follow up (part of #40321):
- [ ] The whole section [jumps](https://d.pr/i/7NHgz0) when you select and deselect the category field (@sirreal)
- [ ] The underline shouldn't be [full width](https://d.pr/i/4V5U9N)
- [ ] The scaling as you enter a longer string is looking great :) Could we wrap it to two lines when it goes too off screen?
